Makefile

```
@@ -547,6 +547,41 @@ kind-image: ## Build cilium and operator images and import them into kind.
 	$(MAKE) kind-image-agent
 	$(MAKE) kind-image-operator
 
+.PHONY: kind-install-full
+kind-install-full:
+	@echo "  INSTALL cilium"
+	# cilium-cli doesn't support idempotent installs, so we uninstall and
+	# reinstall here. https://github.com/cilium/cilium-cli/issues/205
+	-cilium uninstall >/dev/null
+	# cilium-cli's --wait flag doesn't work, so we just force it to run
+	# in the background instead and wait for the resources to be available.
+	# https://github.com/cilium/cilium-cli/issues/1070
+	cilium install \
+		--chart-directory=$(ROOT_DIR)/install/kubernetes/cilium \
+		--helm-values=$(ROOT_DIR)/contrib/testing/kind-values.yaml \
+		--version= \
+		>/dev/null 2>/dev/null &
```

Will
```suggestion
  &>/dev/null &
```
suffice? It's not POSIX but its bash to YOLO

---

Is there a functional difference? I'm not so familiar with POSIX etc. so I'll be relying on mac developers to report if this target needs any further tweaks to work right.

---

Probably `>/dev/null 2>&1` is the standard way to do this:

https://unix.stackexchange.com/questions/590694/posix-compliant-way-to-redirect-stdout-and-stderr-to-a-file

---

```
$ git ap
diff --git a/Makefile b/Makefile
index 31f17c50b39b..20ca5eef98fc 100644
--- a/Makefile
+++ b/Makefile
@@ -560,7 +560,7 @@ kind-install-cilium: kind-ready ## Install a local Cilium version into the clust
                --chart-directory=$(ROOT_DIR)/install/kubernetes/cilium \
                --helm-values=$(ROOT_DIR)/contrib/testing/kind-values.yaml \
                --version= \
-               >/dev/null 2>/dev/null &
+               >/dev/null 2>&1 &
 
 .PHONY: kind-check-cilium
 kind-check-cilium:
```

---

No functional difference, just less lines of code. I think this is a  bashism so any decently new version of bash will use it. 

@chancez may know if this is cool with Mac.

---

Yup that works. You are a man of standards :-D